Taskfile.yml

```diff
@@ -67,22 +67,42 @@ tasks:
   ### -------------------------
   # Primary workflow tasks 
   ### -------------------------
-  all:
-    desc: "do everything necessary after clone (or rebase)"
+  dockerall:
```

Comment:
minor: I think i would prefer that the consistent environment have the name all, and the local environment would be named differently? like `local-all`

Minor as it is clear in the descriptions what these tasks do

Reply:
done